apps/dashboard/config/configuration_singleton.rb

```diff
@@ -291,6 +291,11 @@ def pinned_apps_menu_length
     config.fetch(:pinned_apps_menu_length, 6)
   end
 
+  # Setting terminal functionality in files app
+  def files_app_can_launch_terminal?
+    config.fetch(:files_app_can_launch_terminal, true)
+  end
```

Comment:
I think we can rely on the `to_bool` method we have somewhere in this class?

Also should create some test cases for this too.  It's a super simple API & method, but tests are just safeguards against changes we may make inadvertently. 

The name is ... OK, how about `files_enable_shell_button`?  This may be the toughest part because it'll be hard to change or walk back if we want to rename it. I'm a big fan of Ruby's use of question marks for APIs, but it seems like the config admins type out should be a directive, not a question.

Reply:
Sure thing, missed this comment on my last commit but I'll make the changes, to include a better name.